### PR TITLE
Completed Title component

### PR DESCRIPTION
**Pull request by kavinphab**

### CODE REVIEW

**Comment by kavinphab** [comment]
image

File: src/components/Title.jsx [hunk 1]
```diff
@@ -1,7 +1,15 @@
 import React from "react";
 
-const Title = () => {
-  return <div>Title</div>;
+const Title = ({ title }) => {
+  return (
+    <div className="">
```
Comment (SafeDuck) [inline src/components/Title.jsx:5]: what's this empty div for?

File: src/app/page.js [hunk 2]
```diff
@@ -1,5 +1,7 @@
+import Title from "@/components/Title";
+
 const Home = () => {
-  return <>HISS Website</>;
+  return <Title title="Meet the Team" />;
```
Comment (SafeDuck) [inline src/app/page.js:4]: make sure you remove this before we merge

**Review by SafeDuck — CHANGES_REQUESTED** [review]

File: src/components/Title.jsx [hunk 3]
```diff
@@ -1,7 +1,15 @@
 import React from "react";
 
-const Title = () => {
-  return <div>Title</div>;
+const Title = ({ title }) => {
+  return (
+    <div>
+      <div className="bg-gradient-to-r from-hiss-blue to-hiss-purple mb-2.5 mt-14 h-5 w-1/4 rounded-r-full"></div>
+      <div className="bg-gradient-to-r from-hiss-blue to-hiss-purple h-5 w-1/3 mb-2.5 rounded-r-full"></div>
+      <div className="text-white text-right pr-7 pt-3 font-poppins font-bold text-2xl h-16 w-2/5 bg-gradient-to-r from-hiss-blue to-hiss-purple rounded-r-full">
```
Comment (SafeDuck) [inline src/components/Title.jsx:8]: change the font to text-3xl

**Review by SafeDuck — CHANGES_REQUESTED** [review]

**Review by SafeDuck — APPROVED** [review]

**Pull request merged**